v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/options/MongoDbToBigQueryOptions.java

```diff
@@ -94,4 +95,25 @@ public interface BigQueryWriteOptions extends PipelineOptions, DataflowPipelineO
 
     void setOutputTableSpec(String outputTableSpec);
   }
+
+  /** UDF options. */
+  public interface JavascriptDocumentTransformerOptions extends PipelineOptions {
+    @TemplateParameter.Text(
```

Comment:
`optional = true` is missing from the annotation.

Reply:
updated